index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<title>PoGo GBL Lead Tracker</title>
+
+<body>
+  <div class=doctitle>Pokémon Go Battle League Lead Tracker - Documentation</div>
+  <h1>Jump To:</h1>
+  <p><a href="#great-league">Great League</a></p>
+  <p><a href="#ultra-league">Ultra League</a></p>
+  <p><a href="#master-league">Master League</a></p>
+  <div></div>
+  <h1><span class=get>GET</span>/leagues</h1>
+  <code>
+    <span class=desc>Method</span>: <span class=get>GET</span>
+    <span class=desc>Route</span>: https://placeholder.url/leagues
+    <span class=desc>Headers</span>: Authorization: Bearer YOUR_API_KEY_HERE
```

Comment:
I suspect your API won't implement bearer tokens since we won't be covering them